app/main/views.py

```diff
@@ -48,7 +59,26 @@ def get_iaas():
 
 
 @main.route('/services/<id>')
+@requires_authentication
 def get_service(id):
     service = Service.query.filter(Service.id == id).first_or_404()
 
     return jsonify(id=id, data=service.data)
+
+
+# Test this locally by running the command:
+# curl -i -H "Content-Type: application/json" \
+# -H "Authorization: Bearer myToken" \
+# -X POST \
+# -d @example_listings/SSP-JSON-SCS.json \
+# 127.0.0.1:5000/g6/service/add
+@main.route('/g6/services', methods=['POST'])
```

Comment:
Function level documentation is usually done with [docstrings](https://www.python.org/dev/peps/pep-0257/)